src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+}
+
+export interface NewTodo {
+  title: string,
```

Comment:
Use semicolons instead of commas to separate properties in the NewTodo interface 🚀